src/api.js

```diff
@@ -0,0 +1,43 @@
+const server = 'localhost'
+const port = '5005'
```

Comment:
to make a request use axios according to https://vuejs.org/v2/cookbook/using-axios-to-consume-apis.html.

```
const axios = require('axios');

function getMessages() {
  return axios.get('/messages/states').then(res => res.data.result);
}

export default {
  getMessages
}
```
then in vue.config.js 
```
devServer: {
    proxy: 'http://localhost:5005'
  }
```
basically when you are in production the api and the front are on the same server so you can do a relative call.
then we tell the cli that when developing (npm run serve), we prefer to proxy all requests to localhost:5005.
But this does not work with fetch, that's why i chose axios. 

More info here: https://cli.vuejs.org/config/#devserver-proxy